src/js/components/AppPageComponent.jsx

```diff
@@ -157,7 +157,16 @@ var AppPageComponent = React.createClass({
     });
   },
 
-  onScaleAppError: function (errorMessage) {
+  onScaleAppError: function (errorMessage, statusCode, instances) {
+    if (statusCode === 409) {
+      var appId = this.state.appId;
+      const dialogId = DialogActions.
+        confirm("App scaling failed.Do you want to try forcing?");
+      DialogStore.handleUserResponse(dialogId, function () {
+        AppsActions.scaleApp(appId, instances, true);
+      });
+      return;
```

Comment:
If the user clicks cancel the default actions for the type of dialog will be executed which will just close the dialog. Is this right?